planck-cljs/test/planck/io_test.cljs

```diff
@@ -224,3 +224,17 @@
 (deftest temp-directory-test
   (is (io/file? (io/temp-directory)))
   (is (io/directory? (io/temp-directory))))
+
+(deftest jar-input-stream-test
+  (let [resource-name "META-INF/maven/org.clojure/test.check/pom.properties"
+        resource (io/resource resource-name)
+        target-file-1 (io/temp-file)
+        target-file-2 (io/temp-file)]
+    (io/copy (io/reader resource) target-file-1)
+    (io/copy (io/input-stream resource) target-file-2)
+    (is (= (slurp target-file-1) (slurp target-file-2)))))
+
+(deftest http-input-stream-test
+  (let [target-file (io/temp-file)]
+    (io/copy (io/input-stream "http://planck-repl.org/img/screenshot.png") target-file)
```

Comment:
Instead of the screenshot, I think I'll instead change this to a file under `/releases` (those files are treated as being immutable)